### DISCUSSION TITLE
Suggestions leaking personal information

### DISCUSSION THREAD

**Discussion by darylrobbins** [community/community#19379] [Copilot Conversations] [answered]
```# Calculate the sum of two numbers```

gives me the suggestion
```# Author: Karel Kopecky```

**Comment by melvincwng** [comment, marked as answer]
@darylrobbins, for bugs, kindly report it to [Github Bug Report](https://support.github.com/contact/bug-report) over here.

Hope this clarifies.

**Comment by 42326** [comment]
paid the piedpueper hs rhreTening jf personal PO property or intellectual property. intimidating family of a member is a federal offense. i will contact fcp agency and issue Amber alert watch of my minor daughter if see pictures that resble her posted online zic TOC or you tube. you want messeithan she you mind. im kjaunit@gmail.com not my ten year old daughter
